sql/hive/src/test/scala/org/apache/spark/sql/hive/execution/HiveCommandSuite.scala

```diff
@@ -18,21 +18,32 @@
 package org.apache.spark.sql.hive.execution
 
 import org.apache.spark.sql.{AnalysisException, QueryTest, Row, SaveMode}
+import org.apache.spark.sql.catalyst.TableIdentifier
 import org.apache.spark.sql.catalyst.analysis.NoSuchTableException
-import org.apache.spark.sql.execution.command.CreateDataSourceTableUtils._
+import org.apache.spark.sql.catalyst.catalog.{CatalogStorageFormat, CatalogTable, CatalogTableType}
 import org.apache.spark.sql.hive.test.TestHiveSingleton
 import org.apache.spark.sql.test.SQLTestUtils
+import org.apache.spark.sql.types.StructType
 
 class HiveCommandSuite extends QueryTest with SQLTestUtils with TestHiveSingleton {
   import testImplicits._
 
   protected override def beforeAll(): Unit = {
     super.beforeAll()
-    sql(
-      """
-        |CREATE TABLE parquet_tab1 (c1 INT, c2 STRING)
-        |USING org.apache.spark.sql.parquet.DefaultSource
-      """.stripMargin)
```

Comment:
no way. The CREATE TABLE syntax for data source table doesn't allow specify table properties and we didn't support creating data source table using session catalog before because the hive hacks are in command.